vllm_omni/entrypoints/async_omni.py

```diff
@@ -163,21 +220,7 @@ def _initialize_stages_from_kwargs(self, model: str, kwargs: dict[str, Any]) ->
                             "devices": "0",
```

Comment:
Here may raise the error when sequence_parallel_size > 1, and the device is fixed.